client/src/pages/Catalog.tsx

```diff
@@ -55,24 +40,24 @@ export function Catalog() {
   }
 
   return (
-    <div className="container">
+    <div onClick={closeNavDrawer} className="container">
       <h1 className="font-bold text-5xl text-tertiary my-6">Catalog</h1>
-      <div className="grid grid-cols-2 gap-10 md:grid-cols-3 lg:grid-cols-4 ">
+      <div className="grid grid-cols-2 gap-10 md:grid-cols-3 lg:grid-cols-4 cursor-pointer">
         {products.map((product) => (
-          <div
-            key={product?.productId}
-            className="bg-tertiary rounded flex flex-wrap justify-center p-2 shadow-wrapper">
-            <img
-              className="h-36  md:h-52 lg:h-52 w-full rounded shadow-md"
-              src={product.image}
-              alt="Jordan one"
-            />
-            <h2 className="text-white mr-4">{product.brand}</h2>
-            <h2 className="text-white mr-4">{product.name}</h2>
-            <h3 className="text-white font-bold">
-              {toDollars(product.amount)}
-            </h3>
-          </div>
+          <Link key={product?.productId} to={`/details/${product.productId}`}>
+            <div className="bg-tertiary rounded flex flex-wrap justify-center p-2 shadow-wrapper hover:bg-black">
+              <img
+                className="h-36  md:h-52 lg:h-52 w-full rounded shadow-md"
+                src={product.image}
+                alt="Jordan one"
```

Comment:
Are you sure this is the correct alt tag for all products?
A better way to do this would be to either say `alt="Product image"` or to add a column to your database, such as `altText`, and then you could put `alt={product.altText}`.
Another choice I just realized is you could say ``alt={`${product.name} image`}`` or just `alt={product.name}`.